src/org/jitsi/impl/neomedia/codec/video/h264/JNIEncoder.java

```diff
@@ -658,6 +657,7 @@ else if ("h264.profile".equals(k))
                     //"crf" /* constant quality mode, constant ratefactor */, "0",
                     "intra-refresh", intraRefresh ? "1" : "0",
                     "keyint", Integer.toString(keyint),
+                    "motion-est", "hex", /* possible since FFmpeg 2.7 */
```

Comment:
Will it cause problems(crashes, errors?) when passing that to FFmpeg < 2.7?

Reply:
Have not tested that. It is expected that is simply ignored. Consequently, in FFmpeg 2.6 and lower, the motion-estimation method will not be set.

Reply:
Does this block this Pull Request? Shall I test something? This Pull Request blocks another Pull Request of mine. 

By the way, I wonder why the motion-estimation method is changed at all. Are you able to ask the original contributor why that is done? Why not dropping this all together.

Reply:
It was Lyubomir who did that in the first place, as I remember and he spent time experimenting with some h264 options, but this was long time ago ... I suppose by that time the files were still in jitsi main repo and that was prior ffmpeg 1.0...  So probably that information is already lost/forgotten if there is nothing mentioned in the commit log (which should be imported from the svn repo).

Reply:
Well, I just found your commit https://github.com/jitsi/jitsi/commit/d6dd75f4a0790fd4da3d927939d1f1f07ecc137b back in the year 2008 which added all this.

Reply:
What about simply dropping it?

Reply:
> Well, I just found your commit [jitsi/jitsi@d6dd75f](https://github.com/jitsi/jitsi/commit/d6dd75f4a0790fd4da3d927939d1f1f07ecc137b) back in the year 2008 which added all this.

You see, I told you it is lost, even I don't remember it :) I just remember that by that time I was also looking at some posts of people with recommended settings for h264 encoding for realtime video.

> What about simply dropping it?

As you had already investigated it and saw it is not used and you have more experience with the latest ffmpeg. I'm ok with that.

Reply:
Well, currently it is used. I just tweaks x264 to a state nobody knows whether it provides any benefit. In OpenH264, it is not used and ignored anyway.

Reply:
> Well, currently it is used. I just tweaks x264 to a state nobody knows whether it provides any benefit. In OpenH264, it is not used and ignored anyway.

If it is not used in openh264, let's drop it.

Reply:
Wupps. This was not the commit. Here is the one which changed the method to ‘hex’: https://github.com/jitsi/jitsi/commit/21d1855a1463f138113710702bfc7ca43359970a. Anyway, looks like it was changed because there was a not-so-perfect value before.

Reply:
I dropped it anyway. Now, anyone can blame me.

Reply:
Nobody to blame: The default = `medium` preset of the encoder library x264 (and the wrapper in FFmpeg) is `X264_ME_HEX` anyway. We can drop it. Done with the commit below.